src/mcom/integration.md

```diff
@@ -0,0 +1,24 @@
+---
+ee_only: true
+title: Integration
+---
+
+Integration enables you to add or disable the Order Management System (OMS) integration for Magento Commerce (MC) and view details of the integration.
+
+![Integration]({% link images/images-ee/mcom-integration.png %}){: .zoom}
+_Integration_
+
+## Register the integration
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Integration**.
+1. Click **Register** to register the MCOM (also known as OMS) integration.
+
+## Disable or enable the integration
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Integration**.
+1. Check the status of the integration in the _Status_ row---either `Enabled` or `Disabled`.
+   1. To enable a disabled integration, click **Enable**. A confirmation message appears to inform you the integration has been enabled.
```

Comment:
We don't use "sub numbers", as they are not really supported by the theme. You can nest bullets in a numbered item, bit not other numbers.